### PR TITLE
Songs in common

### PR DESCRIPTION
**Pull request by brandonho2000**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -73,4 +73,9 @@ footer {
   width: 100%;
   height: 2.5rem;    
   font-family: 'Special Elite'; 
-  }
\ No newline at end of file
+  }
+
+  // .container {
```
Comment (sjproctor) [inline app/assets/stylesheets/application.scss]: Remove any commented out code!

**Comment by archae0pteryx** [comment]
I'm not seeing the songs in common code... am i blind? also, need to clean up all the commented out code

**Review by sjproctor — CHANGES_REQUESTED** [review]
Gotta a lotta commented out code here. Can we fix that?

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (brandonho2000) [reply app/assets/stylesheets/application.scss]: fixed!

Comment (archae0pteryx) [reply app/assets/stylesheets/application.scss]: looks like this is still commented?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -73,4 +73,9 @@ footer {
   width: 100%;
   height: 2.5rem;    
   font-family: 'Special Elite';
```
Comment (archae0pteryx) [inline app/assets/stylesheets/application.scss:75]: just an FYI... you should aways define a fallback font if for whatever reason your custom one doesn't load. https://www.w3schools.com/Css/css_font_fallbacks.asp

**Review by archae0pteryx — APPROVED** [review]

**Review by sjproctor — APPROVED** [review]

**Pull request merged**